.setup/data/polls/polls_data.json

```
@@ -16,6 +16,7 @@
             2
         ],
         "release_date": "2021-07-19",
+        "end_time":"2021-08-19",
```

The same spacing issue appears throughout the changes in this file.  Please apply throughout.

```suggestion
        "end_time": "2021-08-19",
```

---

Fixed